lib/nolibc/musl-imported/src/network/h_errno.c

```diff
@@ -0,0 +1,9 @@
+#include <netdb.h>
+
+#undef h_errno
```

Comment:
Is this actually needed? The header above has no `h_errno` define so this will not do anything.